src/mme-app/mmeStates/s1ReleaseWfReleaseAccessBearerResp.cpp

```diff
@@ -64,4 +64,13 @@ void S1ReleaseWfReleaseAccessBearerResp::initialize()
                 actionTable.setNextState(S1ReleaseWfUeCtxtReleaseComp::Instance());
                 eventToActionsMap.insert(pair<uint16_t, ActionTable>(REL_AB_RESP_FROM_SGW, actionTable));
         }
+        {
+                ActionTable actionTable;
+                actionTable.addAction(&ActionHandlers::handle_attach_request);
+                actionTable.addAction(&ActionHandlers::del_session_req);
+                actionTable.addAction(&ActionHandlers::send_s1_rel_cmd_to_ue);
```

Comment:
here also sending S1 release is not required I think. We can clean up locally.

Reply:
Since we have received the attach on a new s1, and from mme perspective, another procedure is still in progress..shouldn't mme release the current s1?

Reply:
Removed s1 release as discussed.

Reply:
compilation is failing after the change. @asreek